test_print_menu.py

```diff
@@ -0,0 +1,86 @@
+from todo import print_menu
+from pytest import fixture
+
+
+
+## Prep
+menu_tasks_naked = [
+    "To-Do List CLI App",
+    "Add task",
+    "List tasks ordered numerically",
+    "List tasks ordered alphabetically",
+    "Delete task",
+    "Add/Update a due date to a task",
+    "Add/Update a tag to a task",
+    "Delete all tasks",
+    "Edit task description",
+    "Show total number of tasks",
+    "Delete a due date to a task",
+    "Delegate task to someone else",
+    "Mark Task Complete",
+    "Quit",
+]
+
+menu_tasks_numbered = ["To-Do List CLI App"]
+for i in range(1, len(menu_tasks_naked)):
+    menu_tasks_numbered.append(f"{i}. {menu_tasks_naked[i]}")
+
+
+
+## Fixtures
+@fixture
+def tasks_count():
+    return len(menu_tasks_naked)
+
+@fixture
+def naked_tasks():
+    return menu_tasks_naked
+
+@fixture
+def numbered_tasks():
+    return menu_tasks_numbered
+
+@fixture
+def entire_menu():
+    return f'\n{'\n'.join(menu_tasks_numbered)}\n'
```

Comment:
Error on line 45 when running the `pytest` command:

```
E     File "/home/penguino/Todo-List-CLI/test_print_menu.py", line 45
E       return f'\n{'\n'.join(menu_tasks_numbered)}\n'
E                     ^
E   SyntaxError: unexpected character after line continuation character
```

This is caused by the single-quoted string literal (`'\n'`) inside the single-quoted string.

## Potential solution

One potential solution is to compute the data, assign it to a variable, then reference the variable in the string:

```python
@fixture
def entire_menu():
    # New variable
    inner = '\n'.join(menu_tasks_numbered)
    return f'\n{inner}\n'
```